modules/gcve-private-cloud/variables.tf

```diff
@@ -0,0 +1,69 @@
+variable project {
```

Comment:
variables names are inconsistently quoted and unquoted. Please quote to align with other variables files in this repo.